internal/provider/resource_model_test.go

```diff
@@ -49,7 +50,9 @@ func TestAcc_ResourceModelImport(t *testing.T) {
 			{
 				Config: testAccResourceModelImport(t, modelName),
 				Check: resource.ComposeTestCheckFunc(
-					resource.TestCheckResourceAttr("juju_model.model", "name", modelName),
+					resource.TestMatchResourceAttr(
+						"juju_model.model", "name", regexp.MustCompile("^"+modelName+"$"),
+					),
```

Comment:
Is there an issue with using `TestCheckResourceAttr`? 

I've been switching uses of `Match` over to `Check` where I've found them - there are a hang-over from the PoC.

Reply:
oh good catch! This is a regression from a merge conflict - I'll put that right